### PR TITLE
Adding an Api for the chatRooms page

### PR DESCRIPTION
**Pull request by Amirk390**
Created chatRoomPosts route, added chatRoomPosts.js file to src/routes folder and added added getChatRoomsPosts function to the postQueries.js.

Reference #12.

### CODE REVIEW

**Comment by Amirk390** [comment]
@idantb101 thanks for the feedback, changes have been made :smile:

File: src/routes/chatRoomRoutes.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const chatRoomsQueries = require('../db/queries/chatRoomsQueries');
+
+const routes = () => {
+  const chatRoomRoutes = express.Router();
```
Comment (Karyum) [inline src/routes/chatRoomRoutes.js]: ok wow there let's stop here. you need to make 1 router file that has `const router = express.Router()` you are making your file structure way too big here, 1 router file 1 controllers folder

File: src/app.js [hunk 2]
```diff
@@ -11,8 +12,10 @@ app.get('/', (req, res) => {
...
 });
 
-app.use('/posts', postsRoutes);
+app.use('/posts', postsRoutes());
 
 app.listen(port, () => {
   console.log(`running on port http://localhost:${port}`);
 });
+
+app.use('/chatrooms', chatRoomRoutes());
```
Comment (Karyum) [inline src/app.js]: also from my comment down below :arrow_down: you will need just 1 `app.use` imagine having a huge app with 30 routes would you `app.use()`  30 times ?!!!

**Comment by Amirk390** [comment]
I'm closing this pull request, 
please continue with #19 

The chatRooms API will need to be rewritten because

**Comment by Karyum** [comment]
@Amirk390 because ? why will it need to be rewritten?

**Comment by Amirk390** [comment]
@Karyum 
> because ? why will it need to be rewritten?

I wrote the code based on a specific idea
after a small discussion with my team, we chose not to have the chat room in that specific way but make it work in a bit different functionality

**Pull request closed without merging**